<!DOCTYPE html>
<html lang="en">
  <head>
      <meta charset="UTF-8">
      <title>New Brain Maps With Unmatched Detail May Change Neuroscience</title>
      <link href="/css/reset.css" rel="stylesheet">
      <link href="/css/styles.css" rel="stylesheet">
  </head>
  <body>
    <header>
    </header>
    <nav>
    </nav>
    <section class="article-container">
    <article>
      <h1>New Brain Maps With Unmatched Detail May Change Neuroscience</h1>
      <figure>
        <img src="https://media.wired.com/photos/5ac7b9bbca846b0c6d91a9f9/master/w_582,c_limit/NeuralPathsArt_2880x1620-2880x1620.jpg" alt="symbolic image of Brain Maps" />
        <figcaption>Olena Shmahalo/Quanta Magazine</figcaption>
      </figure>
      <p>Sitting at the desk in his lower-campus office at Cold Spring Harbor Laboratory, the neuroscientist Tony Zador turned his computer monitor toward me to show off a complicated matrix-style graph. Imagine something that looks like a spreadsheet but instead of numbers it’s filled with colors of varying hues and gradations. Casually, he said: “When I tell people I figured out the connectivity of tens of thousands of neurons and show them this, they just go ‘huh?’ But when I show this to people …” He clicked a button onscreen and a transparent 3-D model of the brain popped up, spinning on its axis, filled with nodes and lines too numerous to count. “They go ‘What the _____!’”</p>
      <p>What Zador showed me was a map of 50,000 neurons in the cerebral cortex of a mouse. It indicated where the cell bodies of every neuron sat and where they sent their long axon branches. A neural map of this size and detail has never been made before. Forgoing the traditional method of brain mapping that involves marking neurons with fluorescence, Zador had taken an unusual approach that drew on the long tradition of molecular biology research at Cold Spring Harbor, on Long Island. He used bits of genomic information to imbue a unique RNA sequence or “bar code” into each individual neuron. He then dissected the brain into cubes like a sheet cake and fed the pieces into a DNA sequencer. The result: a 3-D rendering of 50,000 neurons in the mouse cortex (with as many more to be added soon) mapped with single cell resolution.</p>
      <p>This work, Zador’s magnum opus, is still being refined for publication. But in a paper recently published by Nature, he and his colleagues showed that the technique, called MAPseq (Multiplexed Analysis of Projections by Sequencing), can be used to find new cell types and projection patterns never before observed. The paper also demonstrated that this new high-throughput mapping method is strongly competitive in accuracy with the fluorescent technique, which is the current gold standard but works best with small numbers of neurons.</p>
      <p>The project was born from Zador’s frustration during his “day job” as a neurophysiologist, as he wryly referred to it. He studies auditory decision-making in rodents: how their brain hears sounds, processes the audio information and determines a behavioral output or action. Electrophysiological recordings and the other traditional tools for addressing such questions left the mathematically inclined scientist unsatisfied. The problem, according to Zador, is that we don’t understand enough about the circuitry of the neurons, which is the reason he pursues his “second job” creating tools for imaging the brain.</p>
      <p>The current state of the art for brain mapping is embodied by the Allen Brain Atlas, which was compiled from work in many laboratories over several years at a cost upward of $25 million. The Allen Atlas is what’s known as a bulk connectivity atlas because it traces known subpopulations of neurons and their projections as groups. It has been highly useful for researchers, but it cannot distinguish subtle differences within the groups or neuron subpopulations.</p>
      <p>If we ever want to know how a mouse hears a high-pitched trill, processes that the sound means a refreshing drink reward is available and lays down new memories to recall the treat later, we will need to start with a map or wiring diagram for the brain. In Zador’s view, lack of knowledge about that kind of neural circuitry is partly to blame for why more progress has not been made in the treatment of psychiatric disorders, and why artificial intelligence is still not all that intelligent.</p>
      <p>Justus Kebschull, a Stanford University neuroscientist, an author of the new Nature paper and a former graduate student in Zador’s lab, remarked that doing neuroscience without knowing about the circuitry is like “trying to understand how a computer works by looking at it from the outside, sticking an electrode in and probing what we can find. … Without ever knowing the hard drive is connected to the processor and the USB pod provides input to the whole system, it’s difficult to understand what’s happening.”</p>
      <p>Inspiration for MAPseq struck Zador when he learned of another brain mapping technique called Brainbow. Hailing from the lab of Jeff Lichtman at Harvard University, this method was remarkable in that it genetically labeled up to 200 individual neurons simultaneously using different combinations of fluorescent dyes. The results were a tantalizing, multicolored tableau of neon-colored neurons that displayed, in detail, the complex intermingling of axons and neuron cell bodies. The groundbreaking work gave hope that mapping the connectome—the complete plan of neural connections in the brain—was soon to be a reality. Unfortunately, a limitation of the technique in practice is that through a microscope, experimenters could resolve only about five to 10 distinct colors, which was not enough to penetrate the tangle of neurons in the cortex and map many neurons at once.</p>
      <p>That’s when the lightbulb went on in Zador’s head. He realized that the challenge of the connectome’s huge complexity might be tamed if researchers could harness the increasing speed and dwindling costs of high-throughput genomic sequencing techniques. “It’s what mathematicians call reducing it to a previously solved problem,” he explained.</p>
      <p>In MAPseq, researchers inject an animal with genetically modified viruses that carry a variety of known RNA sequences, or “bar codes.” For a week or more, the viruses multiply inside the animal, filling each neuron with some distinctive combination of those bar codes. When the researchers then cut the brain into sections, the RNA bar codes can help them track individual neurons from slide to slide.</p>
      <p>Zador’s insight led to the new Nature paper, in which his lab and a team at University College London led by the neuroscientist Thomas Mrsic-Flogel used MAPseq to trace the projections of almost 600 neurons in the mouse visual system. (Editor’s note: Zador and Mrsic-Flogel both receive funding from the Simons Foundation, which publishes Quanta.)</p>
      <p>Six hundred neurons is a modest start compared with the tens of millions in the brain of a mouse. But it was ample for the specific purpose the researchers had in mind: They were looking to discern whether there is a structure to the brain’s wiring pattern that might be informative about its function. A currently popular theory is that in the visual cortex, an individual neuron gathers a specific bit of information from the eye—about the edge of an object in the field of view, or a type of movement or spatial orientation, for example. The neuron then sends a signal to a single corresponding area in the brain that specializes in processing that type of information.</p>
      <p>To test this theory, the team first mapped a handful of neurons in mice in the traditional way by inserting a genetically encoded fluorescent dye into the individual cells. Then, with a microscope, they traced how the cells stretched from the primary visual cortex (the brain area that receives input from the eyes) to their endpoints elsewhere in the brain. They found that the neurons’ axons branched out and sent information to many areas simultaneously, overturning the one-to-one mapping theory.</p>
      <p>Next, they asked if there were any patterns to these projections. They used MAPseq to trace the projections of 591 neurons as they branched out and innervated multiple targets. What the team observed was that the distribution of axons was structured: Some neurons always sent axons to areas A, B and C but never to D and E, for example.</p>
      <p>These results suggest the visual system contains a dizzying level of cross-connectivity and that the pattern of those connections is more complicated than a one-to-one mapping. “Higher visual areas don’t just get information that is specifically tailored to them,” Kebschull said. Instead, they share many of the same inputs, “so their computations might be tied to each other.”</p>
      <p>Nevertheless, the fact that certain cells do project to specific areas also means that within the visual cortex there are specialized cells that have not yet been identified. Kebschull said this map is like a blueprint that will enable later researchers to understand what these cells are doing. “MAPseq allows you to map out the hardware. … Once we know the hardware we can start to look at the software, or how the computations happen,” he said.</p>
      <p>MAPseq’s competitive edge in speed and cost for such investigations is considerable: According to Zador, the technique should be able to scale up to handle 100,000 neurons within a week or two for only $10,000 — far faster than traditional mapping would be, at a fraction of the cost.</p>
      <p>Such advantages will make it more feasible to map and compare the neural pathways of large numbers of brains. Studies of conditions such as schizophrenia and autism that are thought to arise from differences in brain wiring have often frustrated researchers because the available tools don’t capture enough details of the neural interconnections. It’s conceivable that researchers will be able to map mouse models of these conditions and compare them with more typical brains, sparking new rounds of research. “A lot of psychiatric disorders are caused by problems at the circuit level,” said Hongkui Zeng, executive director of the structured science division at the Allen Institute for Brain Science. “Connectivity information will tell you where to look.”</p>
      <p>High-throughput mapping also allows scientists to gather lots of neurological data and look for patterns that reflect general principles of how the brain works. “What Tony is doing is looking at the brain in an unbiased way,” said Sreekanth Chalasani, a molecular neurobiologist at the Salk Institute. “Just as the human genome map has provided a scaffolding to test hypotheses and look for patterns in [gene] sequence and function, Tony’s method could do the same” for brain architecture.</p>
      <p>The detailed map of the human genome didn’t immediately explain all the mysteries of how biology works, but it did provide a biomolecular parts list and open the way for a flood of transformative research. Similarly, in its present state of development, MAPseq cannot provide any information about the function or location of the cells it is tagging or show which cells are talking to one another. Yet Zador plans to add this functionality soon. He is also collaborating with scientists studying various parts the brain, such as the neural circuits that underlie fear conditioning.</p>
      <p>“I think there are insights to be derived from connectivity. But just like genomes themselves aren’t interesting, it’s what they enable that is transformative. And that’s why I’m excited,” Zador said. “I’m hopeful it’s going to provide the scaffolding for the next generation of work in the field.”</p>
      <p class="article-imprint">Original story reprinted with permission from Quanta Magazine, an editorially independent publication of the Simons Foundation whose mission is to enhance public understanding of science by covering research developments and trends in mathematics and the physical and life sciences.</p>
    </article>
    </section>
    </section>
    <aside>
    </aside>
    <footer>
    </footer>
  </body>
</html>
